src/main/java/controller/CarComfortController.java

```diff
@@ -0,0 +1,125 @@
+package controller;
+
+import exception.CarRentalException;
+import lombok.RequiredArgsConstructor;
+import models.cars.CarComfort;
+import models.session.Session;
+import services.car.CarComfortService;
+import services.session.SessionService;
+import services.user.UserService;
+
+import java.util.List;
+import java.util.Scanner;
+
+import static exception.ExceptionHandler.handleException;
+
+@RequiredArgsConstructor
+public class CarComfortController {
+    private final CarComfortService carComfortService;
+    private final SessionService sessionService;
+    private final UserService userService;
+    private final Scanner scanner;
+
+    public void programInterface() {
+        handleException(() -> {
+            Session session = sessionService.getActive();
+            while (true) {
+                if (userService.isManager(session.getUserId()) && sessionService.isUserAuthenticated()) {
```

Comment:
I think that `isUserAuthenticated` is redundant here. If we will get the session above it means that user is already authenticated